shuffle/move/examples/main/sources/MessageTests.move

```diff
@@ -0,0 +1,23 @@
+#[test_only]
+module Sender::MessageTests {
+    use Sender::Message;
+    use Std::Signer;
+    use Std::UnitTest;
+    use Std::Vector;
+
+    fun get_account(): signer {
+        Vector::pop_back(&mut UnitTest::create_signers_for_testing(1))
+    }
+
+    #[test]
+    public(script) fun sender_can_set_message() {
+        let account = get_account();
+        Message::set_message_internal(&account,  b"Hello Blockchain");
+
+        let addr = Signer::address_of(&account);
+        assert!(
+          Message::get_message(addr) == b"Hello Blockchain",
+          0
+        );
```

Comment:
Actually, I don't think you need this reference, and you can get rid of set_message_internal  by doing the following I believe:
```suggestion
        let account = get_account();
        let addr = Signer::address_of(&account);
        Message::set_message(account,  b"Hello Blockchain");

        assert!(
          Message::get_message(addr) == b"Hello Blockchain",
          0
        );
```